src/child/spawn-child.ts

```diff
@@ -10,14 +10,17 @@ import { argPrefix, compress } from './argv-payload';
  *   the child process.
  */
 export function callInChild(state: BootstrapState) {
+  const loaderURL = pathToFileURL(require.resolve('../../child-loader.mjs'));
+  loaderURL.searchParams.set(argPrefix, compress(state));
```

Comment:
We can avoid calling `compress(state)` twice to save a few cycles in bootstrapping.